tests/callbacks/test_quantization.py

```diff
@@ -0,0 +1,125 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import copy
+
+import pytest
+import torch
+
+from pytorch_lightning import seed_everything, Trainer
+from pytorch_lightning.callbacks import QuantizationAwareTraining
+from pytorch_lightning.metrics.functional.mean_relative_error import mean_relative_error
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from tests.helpers.zoo_data_models import RegressDataModule, RegressionModel
+
+
+@pytest.mark.parametrize("observe", ['histogram', 'average'])
```

Comment:
do we test for different `qconfig` (string and custom qconfig)?

Reply:
we just test for the strings, the Custom is on user responsibility how he defines it...
in fact with passing string we get the default `qconfig` so yes we have it tested :]